src/helper/wrapper/playwrightWrappers.ts

```diff
@@ -32,4 +32,41 @@ export default class PlaywrightWrapper {
   async waitForUrl(link: string) {
     await this.page.waitForURL(link, { timeout: 30000 });
   }
+
+  async pressKeyboard(button: string) {
+    await this.page.keyboard.press(button);
+  }
+
+  async verifyPageTitle(title: string) {
+    await this.page.waitForURL(process.env[`${title.toUpperCase()}_PAGE_URL`])
+
+    const pageTitle = await this.page.title();
+    expect(pageTitle).toContain(title);
```

Comment:
`expect` is a void method that doesn't return any value. It's better to use it inside step definitions. If you want a method to check if a page title is equal to some string you can just simply return a boolean value and assert operation result in step definitions.